distribution/kernel/carbon-home/repository/conf/axis2/axis2.xml

```diff
@@ -33,6 +33,11 @@
 
     <parameter name="EnableChildFirstClassLoading">${childfirstCL}</parameter>
 
+    <!--This will skip a null value element, if the annotation @IgnoreNullElement is set to that element.

```

Comment:
Do we need this line? Having it from line 37 is enough right?

Reply:
Fixed.